packages/strapi-icons/vite.config.ts

```diff
@@ -18,7 +18,7 @@ export default defineConfig({
       external: (id) => !id.startsWith('.') && !id.startsWith('/'),
       output: {
         dir: 'dist',
-        preserveModules: true,
+        preserveModules: process.env.DTS !== "true",
```

Comment:
You can't make this change because it'll break the current API where people can still do `import { X } from '@strapi/icons/X`

IMO I would just use the dts plugin and remove how it currently works to generate the types, im not sure why it's this way at the moment

Reply:
ok